resources/views/customer/receipt/mail.blade.php

```diff
@@ -1,9 +1,23 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <title>Customer Receipt</title>
-</head>
-<body>
-    Customer Receipt
-</body>
-</html>
\ No newline at end of file
+<style>
+    table{
+        width: 100% !important;
+    }
+    
+
+</style>
+@component('mail::message')
+Good day,
+
+This is your receipt.
+
+
+@component('mail::table')
+Due Date | Sub Total | Discount | Tax | Grand Total | Total Amount Received | Withholding  | Payment Method
+|:--------|:-------------|:-------|:---------|:-------| :-------|:-------|:-------|:-------|
+| {{$receipt['due_date']}} | {{$receipt['grand_total']}} | {{$receipt['discount']}} | {{$receipt['tax']}} | {{$receipt['grand_total']-$receipt['total_amount_received']}} | {{$receipt['total_amount_received']}} | {{$receipt['withholding']}}  | {{$receipt['payment_method']}} |
+@endcomponent
```

Comment:
I think printing a receipt should list down its items, something like this.
![image](https://user-images.githubusercontent.com/32955000/184778859-29d3e283-4891-427c-a003-415aeb960b4c.png)

But instead of following exactly the format as shown in the above image, use the same format as you are creating a new receipt in a form.